### ISSUE TITLE
Slides - support saving as list

### ISSUE THREAD

**Issue by vicariousdrama** [vicariousdrama/cornychat#35]
This feature is only available if window.nostr is present as the user will need to sign the event

Add option to save existing slides as a new Album (this will be a nostr Bookmark Set (kind 30003)) from the room settings.

Prompt for the following to include
- name
- about
- choose image from the set to use as the picture representing the slide album, defaulting to first image

Standard tags are defined at https://github.com/nostr-protocol/nips#standardized-tags, but for sanity, we'll add additional as needed

Include all the images as `r` tags in the following format
```json
["r", "the url", "the caption"]
```

Generate a `d` tag. Assemble the full bookmark set for the album with the user's pubkey who is performing the action

User sign and publish to relays.

**Comment by vicariousdrama** [comment]
This should be a new kind of set
e.g. 30388 Slide Set

**Comment by vicariousdrama** [comment]
Sample using 30003 built with nostree.me, but can't see the lists there.

Can view in listr.lol but can't edit

https://njump.me/naddr1qqkxummnw3ex2efdvy6kvvenxf3rxtfjv33nztf5vc6k2tfe89jkgttyxs6rwerzxyergdtpxvq3gamnwvaz7tmjv4kxz7fwv3sk6atn9e5k7q3qyx6pjypd4r7qh2gysjhvjd9l2km6hnm4amdnjyjw3467fy05rf0qxpqqqp6nx870dng

**Comment by vicariousdrama** [comment]
Sample JSON for posterity

```json
{
  "id": "026adfaaae7fca7d92d4ba2e8d17272ec272e9d577da6118951f68677e6ced5d",
  "pubkey": "21b419102da8fc0ba90484aec934bf55b7abcf75eedb39124e8d75e491f41a5e",
  "created_at": 1711390563,
  "kind": 30003,
  "tags": [
    [
      "title",
      "Slut Dragon Slides"
    ],
    [
      "description",
      "A list about Slut Dragon Slides"
    ],
    [
      "d",
      "nostree-a5f332b3-2dc1-4f5e-99ed-d447db1245a3"
    ],
    [
      "L",
      "me.nostree.ontology"
    ],
    [
      "l",
      "nostree"
    ],
    [
      "l",
      "slut dragon slides"
    ],
    [
      "r",
      "https://i.ibb.co/7GM8yzk/476b7ed9-595f-4e02-b339-27f72187a170.jpg",
      "aa"
    ],
    [
      "r",
      "https://i.ibb.co/HPc1rbV/e3eafc14-ae48-45cc-a665-a207198ca5af.jpg",
      "1234"
    ],
    [
      "r",
      "https://i.ibb.co/31hDwjH/0cb2a30e-e123-40e6-a7eb-f42487ebbab0.jpg",
      "32432432"
    ],
    [
      "r",
      "https://i.ibb.co/9Nmn2Vp/01f84b5c-2f8b-4b51-be4d-db21490590ba.jpg",
      "bassss"
    ],
    [
      "r",
      "https://i.ibb.co/4SbzN0z/7ba3f10d-d35d-4ac0-be55-a8d90269b93e.jpg",
      "bsdsds"
    ],
    [
      "r",
      "https://i.ibb.co/3p18sFr/99493b0c-5bb4-4a95-a3c5-da25a22437dd.jpg",
      "dsadsada"
    ],
    [
      "r",
      "https://i.ibb.co/1TrtqZF/769f1679-4509-4a00-b2de-20da14059fd2.jpg",
      "sadfsacsa"
    ],
    [
      "r",
      "https://i.ibb.co/TvxLtYn/07befe9c-4f5b-42bb-9d27-4d34ad7b2e4f.jpg",
      "dfsdvds"
    ],
    [
      "r",
      "https://i.ibb.co/rQF4DWM/9cc1e2e0-ba0f-46fc-8f57-1101c6e52afb.jpg",
      "slut"
    ],
    [
      "r",
      "https://i.ibb.co/rQF4DWM/9cc1e2e0-ba0f-46fc-8f57-1101c6e52afb.jpg",
      "sadsadsa"
    ],
    [
      "r",
      "https://i.ibb.co/RccRFT0/89676e05-8869-4d78-9c04-3696ba9c7b6a.jpg",
      "asfascsavcsv"
    ],
    [
      "r",
      "https://i.ibb.co/THxcJHW/5921dcad-3d81-4899-b4e6-582a55f47434.jpg",
      "aaaaaa"
    ],
    [
      "r",
      "https://i.ibb.co/RYJXhcH/f3b85828-e325-4eec-8092-934fc90204d3.jpg",
      "ssss"
    ],
    [
      "r",
      "https://i.ibb.co/PWhJ1wn/ef10b497-cbf2-413c-a902-53e16064b78a.jpg",
      "aaaaaaaaaaaaaaaaaaa"
    ],
    [
      "r",
      "https://i.ibb.co/RjjXy87/f90106d7-2908-409f-a58b-a55d23057971.jpg",
      "dfsvdsvdsv"
    ],
    [
      "r",
      "https://i.ibb.co/nQ2WP6R/a78dfebb-4d11-4aca-9d41-a12f9a74b109.jpg",
      "sdfvdgsafd"
    ],
    [
      "r",
      "https://i.ibb.co/BzRNDhs/6812d15f-8d2a-4bbc-add2-6e32fef753c8.jpg",
      "fsdfasgdag"
    ],
    [
      "r",
      "https://imgcdn.stablediffusionweb.com/2024/3/25/5ce76fd1-ab8f-4903-8c22-15d5d8f7c9b5.jpg",
      "\u003e156"
    ],
    [
      "r",
      "https://imgcdn.stablediffusionweb.com/2024/3/25/4999c8c8-1f42-4499-b17c-20d302ed8478.jpg",
      "\u003e160"
    ],
    [
      "r",
      "https://i.ibb.co/6BympPG/5b7c5cde-2b60-40a1-951b-46d29b867c4c.jpg",
      "adult dragons"
    ],
    [
      "r",
      "https://imgcdn.stablediffusionweb.com/2024/3/25/7d6e4966-b998-42ef-94c4-880566bea653.jpg",
      "\u003e168"
    ],
    [
      "r",
      "https://i.ibb.co/Xz5WNMM/64608518-573d-45af-a64d-c4e450098868.jpg",
      "asdsadsadsa"
    ],
    [
      "r",
      "https://imgcdn.stablediffusionweb.com/2024/3/25/9a02b23c-2605-4a33-98c1-1721ddead352.jpg",
      "\u003e176"
    ],
    [
      "r",
      "https://i.ibb.co/z7shVtL/4b07d039-5156-490a-9150-ad085cc20379.jpg",
      "dsafgdsagfdsa"
    ],
    [
      "r",
      "https://imgcdn.stablediffusionweb.com/2024/3/25/fe3a32a0-9fbb-498b-915a-07cd458ef98a.jpg",
      "\u003e184"
    ],
    [
      "r",
      "https://i.ibb.co/JQL2NFR/20ad6571-477e-4755-9c21-0662ee92d894.jpg",
      "wtf"
    ],
    [
      "r",
      "https://imgcdn.stablediffusionweb.com/2024/3/25/5ffa1ea2-b7e7-4aa7-a052-0189c79e65dc.jpg",
      "\u003e192"
    ],
    [
      "r",
      "https://i.ibb.co/s5tpjkh/07b7d63a-35f8-48cd-a6b2-8287f2e99ece.jpg",
      "gfshbsg"
    ],
    [
      "r",
      "https://imgcdn.stablediffusionweb.com/2024/3/25/6912d3a4-7d9a-4ac6-9864-5a85684f3025.jpg",
      "\u003e200"
    ],
    [
      "r",
      "https://imgcdn.stablediffusionweb.com/2024/3/25/7fc94ad2-5cb3-4683-8467-866ef07a5c7d.jpg",
      "\u003e204"
    ],
    [
      "r",
      "https://imgcdn.stablediffusionweb.com/2024/3/25/a8a84982-edfe-4bea-b710-a8d42a9cedde.jpg",
      "\u003e208"
    ],
    [
      "r",
      "https://i.ibb.co/gmhvqpR/13743e4a-6b7b-467f-a047-c92cc0ead0ea.jpg",
      "jkuryjyet"
    ],
    [
      "r",
      "https://i.ibb.co/2vJg2Yy/cce98f6e-c32b-421c-b7f1-983339686c3d.jpg",
      "fbvsdfhs"
    ],
    [
      "t",
      "slut dragon slides"
    ]
  ],
  "content": "",
  "sig": "b5aec4608c0c9c49482650df9a2490daf5cfed267807212fc93d477aa7fed590aeaca00d1bc33a695546f2ea5f2bf29e05945ebeb102ded0c4847eec15f79922"
}
```

**Comment by vicariousdrama** [comment]
Implemented e61bdeb7d37be76627795a60428d25d26afe1c7f 
Deployed to server
